go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/openfaas/faas-cli
```

Comment:
@utsavanand2 i don't think you meant to commit these, perhaps sometime this year we can verify and merge this https://github.com/openfaas/faas-cli/pull/757

Reply:
Yes, these won't be included in the final commit.
I tried rearranging the tests but it was not the issue, as I did include 
`testCase := testCase` inside of the loop that iterates over the tests. I found out that if faas-cli doesn't get a namespace flag or one in the stack yaml, then the Namespace field in the request doesn't switch to `openfaas-fn`, I think that is taken care of on the server. So after handling that in the server, the test is running successfully irrespective of the order. But running all of the tests at once with `go test` is causing it to fail because of the dir in which I'm pulling and creating the stack.yaml, so working on it now.

Reply:
Can you remove the go.mod and go.sum please? We're still using dep.